dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
 >
 > 一般不需要使用此命令。仅当已确定 `task => DM-workers` 映射关系存在，但执行其它命令时仍提示必须刷新它时，你才需要使用此命令。
 
+## dmctl 基本用法--命令模式
+
+命令模式跟交互模式的区别是，执行时只需要在 dmctl 命令后紧跟任务操作执行，任务操作同交互模式参数一致。
+
+> **注意：**
+>
+> 1. 一次 dmctl 命令只能跟一个任务操作
+>
+> 2. 任务操作只能放在 dmctl 命令的最后
```

Comment:
```suggestion
> + 任务操作只能放在 dmctl 命令的最后
```

Reply:
seems strange, Why change `2` to `+`

Reply:
If there is a definite sequence for a list (one item must be listed only before or after another), such as procedures for a task, you can use an ordered list `1. 2. 3.`. If there is no such sequence and every item is just listed altogether, the unordered list must be used instead. By the way, in a note,  a blank line should not appear between items.

Reply:
ok, seems also need change `1.` to `+`

Reply:
https://github.com/pingcap/docs-cn/pull/1901#discussion_r357467803